tests/foreman/destructive/test_katello_certs_check.py

```diff
@@ -105,7 +97,8 @@ def test_positive_install_sat_with_katello_certs(certs_vm_setup):
 
     :CaseAutomation: Automated
     """
-    cert_data, rhel_vm = certs_vm_setup
+    cert_data = certs_setup
+    rhel_vm = sat_ready_rhel
     version = rhel_vm.os_version.major
     rhel_vm.download_repofile(product='satellite', release=settings.server.version.release)
     rhel_vm.register_contenthost(
```

Comment:
Can we update it to use `rhel_vm .register_to_cdn()` instead of running 2 functions, register_contenthost and subscription_manager_attach_pool? when el8 repos will be auto-attached so we could also remove enable_repo step. Non-blocking, as its something outside of scope of the PR :)

Reply:
Updated.